rojak-ui-app/RojakAndroid/app/src/main/java/id/artificialintelligence/rojak/models/PairOfCandidates.java

```diff
@@ -0,0 +1,103 @@
+package id.artificialintelligence.rojak.models;
+
+public class PairOfCandidates {
+    private int id;
+    private String name;
+    private String websiteUrl;
```

Comment:
Thanks for the feedback. I have thought about that too, but since previous models not implemented those yet, so I haven't add that yet.

I guess I'm going to add those `@SerializedName` later today.